google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/AppEngineFlexibleRuntimePanel.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.facet.flexible;
+
+import java.awt.Font;
+
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+
+/**
+ * UI panel containing App Engine flexible runtime information.
+ */
+public class AppEngineFlexibleRuntimePanel {
+
+  private JPanel runtimePanel;
+  private JLabel runtimeLabel;
+  private JLabel runtimeExplanationLabel;
+
+  public AppEngineFlexibleRuntimePanel() {
+    runtimeExplanationLabel.setFont(
```

Comment:
its part of the magic of IJ UX form binding. This class is bound to a GUI form of the same name (with a .form) extension. The platform is responsible for initializing the class properly.